lib/begin.go

```diff
@@ -179,6 +190,9 @@ func (a *ACBuild) beginFromImage(start string, insecure bool) error {
 
 	err = reg.Fetch(app.Name, labels, 0, false)
 	if err != nil {
+		if _, ok := err.(*url.Error); ok {
+			return fmt.Errorf("unknown host when fetching image, local file paths must start with '/' or '.'")
```

Comment:
Ah yep. This also catches the error returned when network is missing. Any idea on how I could only catch when DNS lookup fails? Maybe I should just try to lookup the domain before handing it off to `appc/spec/discovery`, and print the special error message when that fails, but I don't know if it'll be any easier there to differentiate between the network's down and the host doesn't exist.

Maybe a special error message is unnecessary if the documentation / help page for the begin command describe the quirk here?